<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkNvidiaAIAAPreferencePage</class>
 <widget class="QWidget" name="QmitkNvidiaAIAAPreferencePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nvidia AIAA</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="layout">
     <item row="0" column="0">
      <widget class="QLabel" name="serverURILabel">
       <property name="text">
        <string>Server URI</string>
       </property>
       <property name="buddy">
        <cstring>serverURILineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="serverURILineEdit">
       <property name="text">
        <string>http://0.0.0.0:5000/v1</string>
       </property>
      </widget>
     </item>
      <item row="1" column="0">
      <widget class="QLabel" name="serverTimeoutLabel">
       <property name="text">
        <string>Server Timeout (In Sec)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="serverTimeoutSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1200</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="modelFilterLabel">
       <property name="text">
        <string>Filter Models By Label</string>
       </property>
       <property name="buddy">
        <cstring>modelFilterCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="modelFilterCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item> 
     <item row="3" column="0">
      <widget class="QLabel" name="neighborhoodSizeLabel">
       <property name="text">
        <string>SmartPoly Neighborhood Size</string>
       </property>
       <property name="buddy">
        <cstring>neighborhoodSizeSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="neighborhoodSizeSpinBox">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="flipPolyLabel">
       <property name="text">
        <string>Transpose Polygon</string>
       </property>
       <property name="buddy">
        <cstring>flipPolyCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="flipPolyCheckBox">
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
